### ISSUE TITLE
Start-AzureStorageFileCopy - unclear that the DestContext parameter is required

### ISSUE THREAD

**Issue by AspenForester** [Azure/azure-powershell#7852]
Azure.Storage Module Version 4.6.1

When using 
```PowerShell
Start-AzureStorageFileCopy `
    -SrcBlobName $firstBlob.Name `
    -SrcContainerName $firstBlob.ICloudBlob.Container.Name `
    -DestShareName 'destinationshare' `
    -DestFilePath '\' `
    -Context $Context 
```
The help does not make it clear that the DestContext parameter is actually required, even when the (Source)Context and DestContext are the same.

This is what it takes to get the command to copy a blob to a file share:
```PowerShell
Start-AzureStorageFileCopy `
    -SrcBlobName $firstBlob.Name `
    -SrcContainerName $firstBlob.ICloudBlob.Container.Name `
    -DestShareName 'destinationshare' `
    -DestFilePath '\' `
    -Context $Context `
    -DestContext $Context
`

I don't feel that my familiarity with this command is sufficient to submit a pull request on the help.

It would be nice if when DestContext was omitted, an assumption be made that the source and destination contexts are the same.  The parameter is "Context" not "SrcContext", after all!

**Comment by maddieclayton** [comment]
@blueww Can you take a look at this issue?

**Comment by blueww** [comment]
@AspenForester 

Thanks for reporting this!
I can repro the issue. I will try to fix it per our bandwidth and customer requests on this issue.

**Comment by blueww** [comment]
This is already fixed in Az 0.7.0.
When not provide -DestContext, will use -Context as both source and dest Context.
